pipeline/train/configs/opustrainer/teacher.yml

```diff
@@ -3,28 +3,14 @@ datasets:
   backtranslated: <dataset1> # Back-translated data
 
 stages:
-  - start
-  - mid
-  - end
+  - pretrain
   - finetune
 
- # One epoch of only original high-quality data to warm up the model
-start:
-  - original 1.0
-  - until original 1
-
-# Gradually add back-translations to the mix
-# Back-translated corpus can vary a lot in size, so we can try using original to count epochs
-mid:
-  - original 0.7
-  - backtranslated 0.3
-  - until original 1
-
-# Expand back-translations
-end:
+# Back-translated corpus can vary a lot in size, so we can try using original one to count epochs
+pretrain:
```

Comment:
instability was not caused by stages so it's fine to use a simpler schedule

Reply:
Ah, I like this as being simpler.